### ISSUE TITLE
Plugin Geomap: Geocode Widget Not Working in Safari

### ISSUE THREAD

**Issue by jvanulde** [ServiceCanada/wet-boew#2]
When typing the autocomplete list is shown briefly then disappears. When the user stops typing the autocomplete list is not shown. Clicking in the text field after typing sometimes shows the results - but not always. Scrolling through the list of results using arrow keys also scrolls the map.

**Comment by neoinsight** [comment]
Yes, agree, same occurs on iPhone - although oddly not on iPhone emulator in Chrome. Video of iPhone attempt at: https://youtu.be/-OEhkKVbvxg
